.gitlab-ci.yml

```diff
@@ -35,7 +35,9 @@ snitch-cluster-sw:
   artifacts:
     paths:
       - sw/math/include/bits/alltypes.h
+      - target/snitch_cluster/sw/runtime/common/*.h
       - target/snitch_cluster/sw/**/build/*.elf
+      - target/snitch_cluster/sw/runtime/rtl/build/libsnRuntime.a
```

Comment:
Why is this needed? Is there any dependent CI stage job which uses these?

Reply:
There was an issue, but I don't remember anymore why it failed. I will check again

Reply:
I remember the issue again. The [`run.py`](https://github.com/pulp-platform/snitch_cluster/blob/17aa42a7a002c633e91b5948a94b71404d1bd5d9/target/snitch_cluster/sw/apps/blas/gemm/test/run.py#L40) script builds the software including runtime again, but it does not generate the header files again (see [here](https://iis-git.ee.ethz.ch/github-mirror/snitch_cluster/-/jobs/1350489))

Reply:
I see, thanks. I will look into this further, and otherwise we can merge it as is.

Reply:
I reverted the commit to check it, but let me know if I should rerevert it to merge it😉

Reply:
Thanks Tim :) I think we will be able to fix this in https://github.com/pulp-platform/snitch_cluster/pull/137 where we are anyways slightly generalizing the `run.py` script. So no need to revert it now, I think we can get this PR to pass as is once we rebase on the other one.